vault/logical_system_paths.go

```diff
@@ -493,6 +499,77 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
 					Summary: "Unseal the Vault.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							// unseal returns `vault.SealStatusResponse` struct
```

Comment:
I see warnings as a part of `SealStatusResponse` [here](https://github.com/hashicorp/vault/blob/9eaf2671634207d8ecacc69860315f0731fd686a/vault/logical_system.go#LL4725). Should that get handled here too?